docs/source/conf.py

```diff
@@ -23,7 +23,8 @@
 #sys.path.insert(0, os.path.abspath('.'))
 
 if os.environ.get('READTHEDOCS', ''):
-    # RTD doesn't use the Makefile, so re-run autogen_config.py here.
+    # RTD doesn't use the repo's Makefile to build docs. Run autogen_config.py
+    # here to generate rst files from ipynb files.
```

Comment:
Agreed. There's a bunch of activity on projects for conversion now over at RTD, as well as other Sphinx extensions, ipymd, etc. Between all of us, we are aware of a bunch of different tools. I'm going to create a section in my calpoly repo to create both a matrix of conversion tools for the different options md->ipymd, ipymd->md, md in sphinx, md -> rst, etc. I'll ping you, min and matthias for comments/additions/deletions of the options. After that we can run some conversion examples to see what we like best. I really would like to get some consistency across the many repos on tools/packages.